ash-window/src/lib.rs

```diff
@@ -2,7 +2,7 @@
 
 use std::os::raw::c_char;
 
-use ash::{extensions::khr, prelude::*, vk, Entry, Instance};
+use ash::{extensions::ext, extensions::khr, prelude::*, vk, Entry, Instance};
```

Comment:
Then remove:

```rust
#[cfg(any(target_os = "macos", target_os = "ios"))]
use ash::extensions::ext; // portability extensions
```

You can't import the same thing in scope twice.

(But I rather see `cfg()` on the per-platform `const`s then)